### PR TITLE
feature: background order handler for the Node

### PR DESCRIPTION
**Pull request by sshaman1101**
### Node
Added: 
- order handler for executing orders from the Marketplace (search -> propose -> deal)

### CLI:
Added:
- Method for inspecting processing orders from the Node

### CODE REVIEW

File: insonmnia/marketplace/server.go [hunk 1]
```diff
@@ -199,6 +196,10 @@ func (m *Marketplace) CancelOrder(_ context.Context, req *pb.Order) (*pb.Empty,
 	return &pb.Empty{}, nil
 }
 
+func (m *Marketplace) GetProcessing(ctx context.Context, req *pb.Empty) (*pb.GetProcessingReply, error) {
+	return nil, nil
```
Comment (Sokel) [inline insonmnia/marketplace/server.go]: `notImplemented` maybe?

Comment (sshaman1101) [reply insonmnia/marketplace/server.go]: It exists just for matching the Marketplace interface. The Market service itself unable to know anything about processing orders, this method is for the Node only.

Comment (sshaman1101) [reply insonmnia/marketplace/server.go]: But you're right, I must place some note about this into the code.

File: insonmnia/node/market.go [hunk 2]
```diff
@@ -1,18 +1,219 @@
...
+
+const (
+	statusNew uint8 = iota
+	statusSearching
+	statusProposing
+	statusDealing
+	statusDone
+	statusFailed
+
+	orderPoolPeriod = 5 * time.Second
```
Comment (3Hren) [inline insonmnia/node/market.go]: `s/Pool/Poll` maybe?

**Pull request merged**